docs/0-what-is/1-getting-started.mdx

```diff
@@ -5,81 +5,185 @@ sidebar_position: 0
+$ conduit pipelines init
+```
+
+If the `pipelines` directory, you'll notice a new file,
+`pipeline-generator-to-log.yaml` that contains our pipeline's configuration:
+
+```yaml
+version: "2.2"
 pipelines:
-  - id: file-to-file
-    # run pipeline on startup
+  - id: example-pipeline
     status: running
-    description: >
-      Example pipeline reading from file "example.in" and writing into file "example.out".
-
+    name: "generator-to-log"
     connectors:
-      - id: example.in
+      - id: example-source
         type: source
-        # use the built-in file plugin as the source
-        plugin: builtin:file
+        plugin: "generator"
         settings:
-          # read data from example.in
-          path: ./example.in
-
-      - id: example.out
+          # Generate field 'id' that's of the int type
+          # Type: string
+          # Optional
+          format.options.id: 'int'
+          # Generate field 'name' that's of the string type
+          # Type: string
+          # Optional
+          format.options.name: 'string'
+          # The format of the generated payload data (raw, structured, file).
+          # Type: string
+          # Optional
+          format.type: 'structured'
+          # The maximum rate in records per second, at which records are
+          # generated (0 means no rate limit).
+          # Type: float
+          # Optional
+          rate: '1'
+      - id: example-destination
         type: destination
-        # use the built-in file plugin as the destination
-        plugin: builtin:file
-        settings:
-          # write data to example.out
-          path: ./example.out
-          # output the raw payload as a string
-          sdk.record.format: template
-          sdk.record.format.options: '{{ printf "%s" .Payload.After }}'
+        plugin: "log"
 ```
 
-Now start Conduit. You should see a log line saying that the pipeline
-`file-to-file` was created:
+The configuration above tells us some basic information about the pipeline (ID
+and name) and that we want Conduit to start the pipeline automatically. We also
+see a source connector, that uses the `generator` plugin, which is a built-in
+plugin. The connector settings translate into: generate structured data, 1
+record per second. Each generated record should contain an `id` field (type:
+integer) and a `name` field (type: string).
+
+Then we see that we have a destination connector where the data will be written
+to. It uses the `log` plugin, which is a built-in plugin that logs all the
+incoming data. No settings are used here because we're fine with the default
+settings (data will be logged with the `INFO` level).
+ 
+## Run Conduit
+
+With the pipeline configuration being ready, we can run Conduit:
 
 ```shell
-$ ./conduit
-/.../
-2023-04-01T12:34:56+00:00 INF pipeline configs provisioned component=provisioning.Service created=["file-to-file"] deleted=[] pipelines_path=./pipelines
-/.../
+$ conduit --pipelines.path pipelines/
```

Comment:
Needed, because Conduit is installed in another directory.

Reply:
Since we initialized this directory via `conduit init`, shouldn't this path be known? (i.e.: no need to specifying the flag)

Reply:
I thought so too, but nope, it's not. If you install Conduit via a package manager, then you'll use that one binary whenever you want to run Conduit, regardless of what's your working directory. That means that you always need to give it the path. For example, on my machine, Conduit is at `/usr/bin/conduit`. So we either need the `pipelines.path` flag or the config path.

Another option is to change Conduit so that it always scans the current working directory. But I'm not sure that's the best way when Conduit is working as a service in a VM or in a container, when I think it should always use a known path.